userspace/libscap/CMakeLists.txt

```diff
@@ -142,8 +142,13 @@ endif()
 option(CREATE_TEST_TARGETS "Enable make-targets for unit testing" ON)
 
 if(CREATE_TEST_TARGETS AND NOT WIN32)
-		# Add unit test directories
-		add_subdirectory(test)
+	# Add engine only used for testing
+	add_definitions(-DHAS_ENGINE_TEST_INPUT)
```

Comment:
Minor note: i think these tests can be enabled on windows also, right?

Reply:
we can do this in another PR maybe :thinking:

Reply:
Completely agree!